auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -202,24 +201,6 @@ public void shouldLoginWithMFAOTPCode() throws Exception {
         assertThat(body, not(hasKey("connection")));
     }
 
-    @Test
-    public void shouldLoginWithUserAndPassword() throws Exception {
-        mockAPI.willReturnSuccessfulLogin();
-
-        final MockAuthenticationCallback<Credentials> callback = new MockAuthenticationCallback<>();
-
-        client.login(SUPPORT_AUTH0_COM, "voidpassword", MY_CONNECTION)
-                .start(callback);
-
-        assertThat(callback, hasPayloadOfType(Credentials.class));
-
-        final RecordedRequest request = mockAPI.takeRequest();
-        assertThat(request.getHeader("Accept-Language"), is(getDefaultLocale()));
-        Map<String, String> body = bodyFromRequest(request);
-        assertThat(body, hasEntry("connection", MY_CONNECTION));
-        assertThat(body, not(hasKey("realm")));
-    }
-
     @Test
     public void shouldLoginWithUserAndPasswordSync() throws Exception {
```

Comment:
I'll add a card to track these Async/Sync test variations. We should definitely cover them but on the networking client class (generic) rather than for every available endpoint. Tracking in SDK-2203.